validator/project_validator_test.go

```diff
@@ -649,22 +649,74 @@ buildvariants:
 }
 
 func TestCheckRequestersForTaskDependencies(t *testing.T) {
```

Comment:
Since this is only tests for the case where it should emit a warning, can we write a test for the success case, like having a `patch_only: true` task + a `patch_only: true` dependency?

Reply:
Sure, added